### PR TITLE
Add Fabric L2 Unicast VLAN Test

### PR DESCRIPTION
**Pull request by TakeshiTseng**

### CODE REVIEW

File: tests/ptf/fabric.ptf/test.py [hunk 1]
```diff
@@ -262,6 +262,38 @@ def runTest(self):
...
+        self.add_next_hop(10, self.port1)
+        self.add_next_hop(20, self.port2)
+        # pops VLAN on port 2 since port 2 is an untagged port
+        self.set_egress_vlan_pop(self.port2, vlan_id)
+
+        pkt_1to2 = testutils.simple_tcp_packet(
+            eth_src=HOST1_MAC, eth_dst=HOST2_MAC, dl_vlan_enable=True, vlan_vid=vlan_id, ip_ttl=64)
+        exp_pkt_1to2 = testutils.simple_tcp_packet(
+            eth_src=HOST1_MAC, eth_dst=HOST2_MAC,
+            ip_ttl=63, pktlen=96) # packet length will decrease
```
Comment (ccascone) [inline tests/ptf/fabric.ptf/test.py]: Why do you need to specify `pktlen` here? I might be wrong, but if the VLAN header is removed from `pkt_1to2`, I expect its length to be the same as a packet returned by `testutils.simple_tcp_packet()` without setting a VLAN ID.

Comment (TakeshiTseng) [reply tests/ptf/fabric.ptf/test.py]: the simple_tcp_packet generates payload according to pktlen(default is 100) and total length of headers

So the payload length of `pkt_1to2` will be 41, and `exp_pkt_1to2` will be 45 if we don't change the pktlen

Comment (ccascone) [reply tests/ptf/fabric.ptf/test.py]: Got it, thanks!

**Comment by ccascone** [comment]
@TakeshiTseng can this be merged?

**Comment by TakeshiTseng** [comment]
@ccascone Yes, I think we can merge it

**Pull request merged**